student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    @classroom = classroom
+    @name = name
+    super(age, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] Name should be included in the `super` call because we also want to get the name from the parent class. hint;

```suggestion
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    @classroom = classroom
    super(age, name, parent_permission)
  end
```